app/grandchallenge/algorithms/views.py

```diff
@@ -617,7 +617,7 @@ def columns(self):
             Column(title="Result", sort_field="inputs__image__name"),
             Column(title="Comment", sort_field="comment"),
             Column(title="Visibility", sort_field="public"),
-            Column(title="Viewer", sort_field="inputs__image__files__file"),
+            Column(title="Viewer", sort_field="title"),
```

Comment:
The sort field here is the one that is used on the model, in this case an Algorithm `Job`. A job does not have a `title` field, so in this case this will break the sorting of the table when the user clicks on Viewer. IMO this change should be reverted.

Reply:
ah, ok. But a strange sorting then?

Reply:
I guess! I don't know what the user would expect when sorting by Viewer. Maybe `pk` makes more sense, or `inputs__image__name`?

Reply:
> 

Well, I thought I would try that out, and it looks like something really strange happens when sorting by viewer, see video. The list shows all results (14 out of 14), but sorting on viewer seems to change the results (just look at the number of results that have failed).

https://user-images.githubusercontent.com/51329283/143855916-53f5a53e-8f49-4671-90e3-2a35ab00ba5a.mp4

Reply:
Weird, maybe you start to get duplicates when sorting by `inputs__image__files__file` as there are many files per image, so a `distinct` is missing somewhere. Maybe `status` is the best thing to go with?